subfun/BIDS_Results.m

```diff
@@ -0,0 +1,142 @@
+function BIDS_Results(FFXSmoothing, ConSmoothing, opt, isMVPA)
+% This scripts computes the results for a series of contrast that can be
```

Comment:
How is that different from the FFX or RFX script which should compute the same thing?

Reply:
Sorry the help section is really unhelpful here: this script actually does all the stuff that the user one would need to do when clicking `Results` on the GUI interface. 
The other nice thing is that this allows to easily display the same results for all subjects of group rather than having to do that by hand.